docs/docs/themes/shadowing.md

```diff
@@ -14,29 +14,28 @@ If you’ve installed `gatsby-theme-blog` you’ll notice that it renders a `Bio
 
 You can inspect `gatsby-theme-blog`'s file structure to determine the file path for the file you want to shadow.
 
-```text:title="tree gatsby-theme-blog"
+```text:title=gatsby-theme-blog
+├── gatsby-browser.js
 ├── gatsby-config.js
 ├── gatsby-node.js
 └── src
     ├── components
     │   ├── bio-content.js
     │   ├── bio.js // highlight-line
     │   ├── header.js
-    │   ├── headings.js
     │   ├── home-footer.js
     │   ├── layout.js
+    │   ├── post-date.js
     │   ├── post-footer.js
+    │   ├── post-hero.js
+    │   ├── post-link.js
+    │   ├── post-list.js
+    │   ├── post-title.js
     │   ├── post.js
     │   ├── posts.js
-    │   ├── seo.js
-    │   └── switch.js
+    │   └── seo.js
     ├── gatsby-plugin-theme-ui
-    │   ├── colors.js
-    │   ├── components.js
-    │   ├── index.js
-    │   ├── prism.js
-    │   ├── styles.js
-    │   └── typography.js
+    │   └── components.js
     └── templates
```

Comment:
Is this still correct? I may be looking at the wrong state but it looks like this `templates` directory is deeper in 2.0?

Reply:
This is interesting since there aren't templates in blog theme, it's in core. I'll look more closely.